scripts/data/common/killed-in-gaza/reconcile_lists.ts

```diff
@@ -975,6 +975,5 @@ async function reconcileCSVs(
 
 reconcileCSVs(
   path.resolve(__dirname, "data/raw.csv"),
-  "scripts/data/common/killed-in-gaza/output/20240501.csv"
-  // path.resolve(__dirname, "data/raw-v2.csv")
+  "scripts/data/common/killed-in-gaza/output/20240501_input_reconcile.csv"
```

Comment:
just noticed this, should this be `20240726_input_reconcile` instead? based on the files you shared with me?

Reply:
though i ran it with that file and after some fixes to the script...

(ie: `someAgeAndNameChanges: mergedRecordConflictChangeDist.age_name?.slice(0,10)` and same optional chain for the line below it)

...not seeing any changes so I'm a bit confused (stats say no added records and no removed records)

here's a specific example of a weird record in the updated `raw.csv` on this branch that i'm trying to figure out:

`804678968` shows up in the CSVs you shared as the same 21yr old female, DOB 2002-02-02

but in the raw.csv diff that record is removed and added without age and the DOB changed to 2000-02-02

(i ran `git diff 20240726 main -- scripts/data/common/killed-in-gaza/data/raw.csv` locally with latest branch states)

Reply:
oh whoops.. i need to reset raw.csv before I run it again 🤦🏻 ... brb

Reply:
i think it might be time to retire the reconcile script as you suggested in discord and instead just accept the ministry lists as we receive them with some cleaning (as you'd done in the python script to normalize the records)

the main thing we can still do is diff the record changes so we can publish an update post that details what's changed between list updates (and to have some confidence our cleaning is accurate), let me try to script that quick stats diff quick